### ISSUE TITLE
multi parts VOD recover 

### ISSUE THREAD

**Issue by kai933** [TwitchRecover/TwitchRecover#436]
the following stream was/is splitted into 4 single vods, due to short interruptions on the part of the streamer.

https://twitchtracker.com/.......

these have a length of 1min, 27min, 17min and 1h15min.

when i use vodrecover from ItIckeYd it only finds 159 segments.

these segments are from the 2nd vod with 27min playing time.

how can i find the other segments of the remaining vods?
statistic sites like sullygnome and twitchtracker etc see the single vods as one 2h17min stream.

yes the vods are still online on the streamers twitch channel at the time i am writing this and i have already backed them up, but they will definitely become hidden or deleted.
But should I not have the luck in a future stream that they are still online I would like to know if there is a way to get the fragments of the other single vods.

Thanks

**Comment by ghost** [comment]
@kai933 Streamscharts

If im trying the right vods... None of them exist.

**Comment by kai933** [comment]
yes as suspected the vods were deleted and not just set hidden, but I had already backed them up =)

but thanks for streamcharts, I have not yet known and seems much better to recognize when there were short interruptions in the stream which result in new vods.

thanks
